src/client/metadataApiDeploy.ts

```diff
@@ -464,3 +421,46 @@ export interface ScopedPostDeploy {
   deployResult: DeployResult;
   orgId: string;
 }
+
+const getState = (message: DeployMessage): ComponentStatus => {
+  if (message.created === 'true' || message.created === true) {
+    return ComponentStatus.Created;
+  } else if (message.changed === 'true' || message.changed === true) {
+    return ComponentStatus.Changed;
+  } else if (message.deleted === 'true' || message.deleted === true) {
+    return ComponentStatus.Deleted;
+  } else if (message.success === 'false' || message.success === false) {
+    return ComponentStatus.Failed;
+  }
+  return ComponentStatus.Unchanged;
+};
+
+/**
+ * Fix any issues with the deploy message returned by the api.
+ * TODO: remove cases if fixes are made in the api.
```

Comment:
just checking if this is still a valid todo

Reply:
I don't know of a good way to find out.

Reply:
the Settings and lwc bundle namespace are pretty recent because I remember those bugs!